### PR TITLE
Support inline text formatting in table cells.

### LINKED ISSUE #634: Table cell markdown translator to support rich styling.
## Summary
Wysiwym support rich text bold/italic etc in table cells. But markdown translator needs to be fixed to support that as well.

Currently it is not possible to have multiple paragraphs/blocks in table cell, it supports only inline content. If markdown supports multiple blocks in table cell we can add that also.

### PR DESCRIPTION
**Pull request by jpuri**
PR has changes to support inline style in table cells.

Fix #634

### CODE REVIEW

**Comment by ncphillips** [comment]
Just a note, those unused dependency warnings are not an issue. Those packages should be ignored. I'm updating the config in #665

File: packages/@tinacms/fields/src/Wysiwyg/Translator/MarkdownTranslator/commonmark/__tests__/table_with_marks/output.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 | First Header | Second Header |
 | --- | --- |
-| Content Cell | Content Cell |
-| Content Cell | Content Cell |
+| _Content Cell_ | **Content Cell** |
+| [Content Cell](/test) | `Content Cell` |
```
Comment (jpuri) [inline packages/@tinacms/fields/src/Wysiwyg/Translator/MarkdownTranslator/commonmark/__tests__/table_with_marks/output.md]: Marks should now work in tables.

File: packages/@tinacms/fields/src/Wysiwyg/Translator/MarkdownTranslator/commonmark/tokens.ts [hunk 2]
```diff
@@ -231,21 +237,9 @@ const NODES: Nodes = {
...
-    node.forEach((n, _, _i) => {
-      const lines = n.text ? n.text.split('\n') : []
-      for (let i = 0; i < lines.length; i++) {
-        const startOfLine = state.atBlank() || state.closed
-        state.write()
-        state.out += state.esc(lines[i], startOfLine).replace(/[\|]/g, '\\$&')
-        if (i != lines.length - 1) state.out += '\n'
-      }
-    })
+    state.renderInline(node, (str: String) => str.replace(/[\|]/g, '\\$&'))
```
Comment (jpuri) [inline packages/@tinacms/fields/src/Wysiwyg/Translator/MarkdownTranslator/commonmark/tokens.ts]: From table cell we pass down this escape function to escape pipes.

**Comment by jpuri** [comment]
Fixes: https://github.com/tinacms/tinacms/pull/664

Fixes: https://github.com/tinacms/tinacms/issues/634

**Pull request merged**